pkg/commands/create/create.go

```diff
@@ -0,0 +1,181 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package create
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"sigs.k8s.io/kustomize/v3/k8sdeps/kunstruct"
```

Comment:
We don't allow the dependency on `k8sdeps` at this level.

Reply:
Ah, I notice now that a kunstructured factory is able to be threaded through from the root command, will update. I assume it's fine for the tests to depend on k8sdeps?